Doc/library/sqlite3.rst

```diff
@@ -1235,17 +1302,22 @@ Connection objects
 
    .. attribute:: isolation_level
 
-      This attribute controls the :ref:`transaction handling
-      <sqlite3-controlling-transactions>` performed by :mod:`!sqlite3`.
+      Legacy control of the :ref:`transaction handling
+      <sqlite3-transaction-control-isolation-level>` performed by :mod:`!sqlite3`.
       If set to ``None``, transactions are never implicitly opened.
       If set to one of ``"DEFERRED"``, ``"IMMEDIATE"``, or ``"EXCLUSIVE"``,
       corresponding to the underlying `SQLite transaction behaviour`_,
       implicit :ref:`transaction management
```

Comment:
```suggestion
      :ref:`implicit transaction management
```

Clarify that the link target includes specific discussion of this, rather than transaction management more generally